### PR TITLE
Added kubectl completion command for fish shell in kubect cheatsheet docmentation

### PR DESCRIPTION
**Pull request by utkarsh-singh1**
Description:
Added kubectl completion for fish shell

Page to Update:
https://kubernetes.io/docs/reference/kubectl/cheatsheet/#kubectl-context-and-configuration

### CODE REVIEW

File: content/en/docs/reference/kubectl/cheatsheet.md [hunk 1]
```diff
@@ -39,6 +39,15 @@ complete -o default -F __start_kubectl k
 source <(kubectl completion zsh)  # set up autocomplete in zsh into the current shell
 echo '[[ $commands[kubectl] ]] && source <(kubectl completion zsh)' >> ~/.zshrc # add autocomplete permanently to your zsh shell
 ```
+
+### FISH
+
+Require kubectl version 1.23 or more.
```
Comment (tengqm) [inline content/en/docs/reference/kubectl/cheatsheet.md:45]:
```suggestion
Require kubectl version 1.23 or above.
```

**Comment by utkarsh-singh1** [comment]
Hi @tengqm , I have updated the necessary changes.

**Comment by dipesh-rawat** [comment]
Hi @utkarsh-singh1,

I see the Netlify build timed out.

You can try following to trigger Netlify rebuild without losing `lgtm` label.

```
git reset
git commit --amend --no-edit
git push --force-with-lease
```

**Review by ydFu — APPROVED** [review]
/lgtm

**Comment by utkarsh-singh1** [comment]
Hi @ydFu and @dipesh-rawat , thanks for your help but - 

`git push --force-with-lease`

gives an error

```bash
fatal: The current branch update-docs-reference-kubectl-cheatsheet-md has no upstream branch.
To push the current branch and set the remote as upstream, use

    git push --set-upstream origin update-docs-reference-kubectl-cheatsheet-md
```

**Comment by dipesh-rawat** [comment]
@utkarsh-singh1 Advancing this PR towards a merge would be beneficial.
It could be worthwhile to consider rebasing this PR with the main branch (once again) and check if that resolves the build errors.

**Comment by kbhawkey** [comment]
👋 @utkarsh-singh1 .
Are you able to send in another commit (empty or otherwise) to trigger a rebuild?
The changes LGTM.
Thanks

**Comment by divya-mohan0209** [comment]
Hello @utkarsh-singh1 , thank you for your PR. However, due to unresponsiveness, I will be closing off this PR. Please note that if you wish to resume work on this, you can reopen the PR with a /reopen command.
/close

**Comment by utkarsh-singh1** [comment]
Hi @dipesh-rawat , @reylejano , @kbhawkey , just push the new PR and it seems to resolve build issues please review this PR.

**Pull request merged**